### PR TITLE
Localized Cluster.md

### PR DESCRIPTION
**Pull request by abh139**
I have Localized the Cluster.md file under the issue #771 
Please check.

### CODE REVIEW

File: content/hi/Cluster.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: समूह
```
[the remark this answers is not in the document]
Comment (abh139) [reply content/hi/Cluster.md]: check now

**Review by anubha-v-ardhan — COMMENTED** [review]
@abh139 This PR has unwanted commits adding changes on LOCALIZATION.md which is out of the scope of Hindi localization. Can you fix this please

**Comment by abh139** [comment]
> @abh139 This PR has unwanted commits adding changes on LOCALIZATION.md which is out of the scope of Hindi localization. Can you fix this please

Check now

**Review by iamNoah1 — COMMENTED** [review]
You have to base your changes on dev-hi. Seems that you have based on main.

**Pull request closed without merging**